hazelcast-jet-core/src/main/java/com/hazelcast/jet/Processors.java

```diff
@@ -90,7 +90,55 @@ public static ProcessorMetaSupplier writeMap(@Nonnull String mapName) {
      */
     @Nonnull
     public static ProcessorMetaSupplier writeMap(@Nonnull String mapName, @Nonnull ClientConfig clientConfig) {
-        return WriteIMapP.supplier(mapName, clientConfig);
+        return WriteBufferedP.writeMap(mapName, clientConfig);
+    }
+
+    /**
+     * Returns a meta-supplier of processors that will fetch entries from the
+     * Hazelcast {@code ICache} with the specified name. The processors will only
+     * access data local to the member and, if {@code localParallelism} for the
+     * vertex is above one, processors will divide the labor within the member
+     * so that each one gets a subset of all local partitions to read.
+     * <p>
+     * The number of Hazelcast partitions should be configured to at least
+     * {@code localParallelism * clusterSize}, otherwise some processors will have
+     * no partitions assigned to them.
+     * <p>
+     * If the underlying cache is concurrently being modified, there are no guarantees
+     * given with respect to missing or duplicate items.
+     */
+    @Nonnull
+    public static ProcessorMetaSupplier readCache(@Nonnull String cacheName) {
+        return ReadWithPartitionIteratorP.readCache(cacheName);
+    }
+
+    /**
+     * Returns a meta-supplier of processors that will fetch entries from a
+     * Hazelcast {@code ICache} in a remote cluster.
```

Comment:
can we mention that it emits entries of type `Cache.Entry`/`Map.Entry` in the javadocs? for both local and remote, and also for Map as well.